File: Dobson/DobsonLibrary/Domain/Modifier.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DobsonLibrary.Domain
{
    public class Modifier
    {
        public int ModifierId { get; set; }
        public string ModifierString { get; set; }
        public int Rank { get; set; }
        public int IndividualId { get; set; }
        
        public Modifier(int modifierId, string modifierString, int rank, int individualId)
        {
            this.ModifierId = modifierId;
            this.Rank = rank;
            this.ModifierString = modifierString;
            this.IndividualId = individualId;
           }


    }
}

File: trunk/ ufmorpheus --username gycordero/OWLMorpheus/src/db/Modifier.java
package db;

public class Modifier {
	private int modifierid;
	private String modifierstring;
	private int rank;
	public Modifier(int modifierid, String modifierstring, int rank){
		this.modifierid = modifierid;
		this.modifierstring = modifierstring;
		this.rank = rank;
	}
	public int getModifierid() {
		return modifierid;
	}
	public String getModifierstring() {
		return modifierstring.replaceAll(" ", "_");
	}
	public int getRank() {
		return rank;
	}
}
